spec/migration/migrators/registration_period_spec.rb

```diff
@@ -0,0 +1,57 @@
+RSpec.describe Migrators::RegistrationPeriod do
+  describe '.record_count' do
+    it 'returns the count of cohorts' do
+      FactoryBot.create_list(:migration_cohort, 2, :with_sequential_start_year)
+      expect(described_class.record_count).to eq(2)
+    end
+  end
+
+  describe '.model' do
+    it 'returns :registration_period' do
+      expect(described_class.model).to eq(:registration_period)
+    end
+  end
+
+  describe '.cohorts' do
+    it 'returns all cohorts' do
+      cohort = FactoryBot.create(:migration_cohort)
+      expect(described_class.cohorts).to include(cohort)
+    end
+  end
+
+  describe '.reset!' do
+    before do
+      FactoryBot.create(:registration_period)
+      allow(Rails.application.config).to receive(:enable_migration_testing).and_return(enabled_migration_testing)
+    end
+
+    context 'when migration testing is enabled' do
+      let(:enabled_migration_testing) { true }
```

Comment:
Might be a tiny bit clearer here to mirror the setting in the variable name and call it `enable_migration_testing`.

Reply:
This was meant to be `enable_migration_testing`. Cheers!